src/odemis/gui/comp/overlay/world.py

```diff
@@ -1255,31 +1264,136 @@ def _view_to_phys(self):
             self.p_end_pos = self.cnvs.view_to_phys(self.v_end_pos, offset)
             self._calc_edges()
 
-    @abstractmethod
+    def _phys_to_view(self):
+        offset = self.cnvs.get_half_buffer_size()
```

Comment:
how is `self.cnvs` set? I might be missing it, but I do not see `self.cnvs = cnvs` in the init